setup.py

```diff
@@ -0,0 +1,24 @@
+
+from setuptools import setup, find_packages
+
+setup(
+    name='agentless-system-crawler',
+    version='0.0.1.dev',
+    #description='',
+    #long_description=long_description,
+    author='IBM',
+    #author_email='',
+    license='apache2',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: System Administrators',
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python :: 2.7',
+    ],
+    packages = find_packages(),
+    install_requires=['psutil','netifaces',],
```

Comment:
Generic versions go in `setup.py`, specific pinned versions go in the `requirements.txt`.